src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -216,6 +218,7 @@
  :sign-request-failed
  (fn [{{:keys [view-id modal] :as db} :db} [_ {:keys [id method error_code error_message]}]]
    (let [send-transaction (get-in db [:wallet :send-transaction])]
+     (log/debug "igorm -> send-transaction" send-transaction)
```

Comment:
looks good, can you just remove the debug logs?
(or did you want to merge them in?)

Reply:
oops, not only @oskarth is distracted today 😆

Reply:
fixed